app/controllers/users.js

```diff
@@ -87,3 +89,44 @@ exports.get = (req, res, next) => {
       next(errors.databaseError(err));
     });
 };
+
+exports.list = (req, res, next) => {
+  let query = {
+    page_number: 1,
+    page_size: 100
+  };
+  if (req.body !== null && Object.keys(req.body).length !== 0) {
```

Comment:
why this?

Reply:
for some reason req.body = {} when nothing is sent